clientd3d/statgame.c

```diff
@@ -72,9 +72,10 @@ void GameExit(void)
    // Can't close modules here, because WM_DESTROY messages sent to dialogs causes crash.
    // Modules are freed elsewhere.
 
-   SoundClose();
    MusicAbort();
-
+   SoundClose();
+   MusicClose();
```

Comment:
I'm curious about this change.  It's not called only when exiting the client, but also when exiting back to the main menu.  With this change, do sounds in progress continue instead?